packages/sdk/src/rtl/transport.js

```diff
@@ -0,0 +1,320 @@
+"use strict";
```

Comment:
As per discussion. Should, probably not be checked in

Reply:
thanks, removed.